### PR TITLE
[O2B-593] Add LHC Fill details page

### PR DESCRIPTION
**Pull request by martinboulais**
#### I have a JIRA ticket
- [x] branch and/or PR name(s) include(s) JIRA ID
- [x] issue has "Fix version" assigned
- [x] issue "Status" is set to "In review"
- [x] PR labels are selected
- [x] `CHANGELOG.md` and `database/CHANGELOG.md` files were updated

Includes the following refactoring:

- Reformat the detail page utility ("format")
- Improve error display utility to handle multiple errors at once
- renamed route lhcFill-overview to lhc-fill-overview

### CODE REVIEW

**Review by JikHuijberts — APPROVED** [review]

File: lib/public/views/LhcFills/Detail/lhcFillDetailsComponent.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import { h } from '/js/src/index.js';
+import spinner from '../../../components/common/spinner.js';
+import table from '../../../components/Table/index.js';
+import errorAlert from '../../../components/common/errorAlert.js';
+import { runsActiveColumns } from '../../Runs/ActiveColumns/runsActiveColumns.js';
+import { lhcFillDisplayConfiguration } from './lhcFillDisplayConfiguration.js';
+import { detailsGrid } from '../../../components/Detail/detailsGrid.js';
+
+/**
+ * Component displaying information about a run
```
Comment (graduta) [inline lib/public/views/LhcFills/Detail/lhcFillDetailsComponent.js:23]: Please update the documentation accordingly

**Review by graduta — CHANGES_REQUESTED** [review]
As a general note, I think we should move away from using `fillId` on the frontend because the real name of the variable and how it is being stored it is `fillNumber`

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ All notable changes to this project will be documented in this file. See [standa
 ## [0.34.0](https://github.com/AliceO2Group/Bookkeeping/releases/tag/%40aliceo2%2Fbookkeeping%400.34.0)
 * Notable changes for users:
   * Run tags update is now integrated in the global run update
+  * A detail page has been created for LHC fills and is accessible from LHC fills overview, run details and runs
```
[the remark this answers is not in the document]
Comment (martinboulais) [reply CHANGELOG.md]: I moved it

File: lib/public/views/LhcFills/Detail/lhcFillDetailsComponent.js (hunk 1 above)
Comment (martinboulais) [reply lib/public/views/LhcFills/Detail/lhcFillDetailsComponent.js]: Done

File: lib/database/seeders/20220503120937-lhc-fills.js [hunk 3]
```diff
@@ -45,16 +45,22 @@ module.exports = {
         },
         {
             fill_number: 4,
+            beam_type: 'p-p',
```
[the remark this answers is not in the document]
Comment (martinboulais) [reply lib/database/seeders/20220503120937-lhc-fills.js:48]: Done

**Review by graduta — APPROVED** [review]

**Pull request merged**